website/pages/chapters/assessing_performance.js

```diff
@@ -242,26 +314,40 @@ export default function AssessingPerformance() {
         </ul>
 
         <p>
-          So even though value we really care about is the <em>true error</em>,
-          we will use the error on the test set as a stand-in for that value. We
-          call the error made by the model on the test set the <b>test error</b>
-          , which we can compute. In the case of regression using RSS as the
-          loss function
-          <MarginNote counter={marginNoteCounter} id="new-notation">
+          So even though the value we really care about is the{" "}
+          <em>true error</em>, we will use this error computed from the test set
+          as a stand-in for that value. We call the error made by the model on
+          the test set the <b>test error</b>, which we <em>can</em> compute. In
+          the case of regression using RSS as the loss function
+          <MarginNote counter={marginNoteCounter}>
             📝 <em>Notation:</em> We use a new notation for{" "}
             <IM math={`\\hat{f}`} /> to signify that it is the predictor defined
             by our estimates for the coefficients <IM math={`\\hat{w}`} /> by
             saying that <IM math={`\\hat{f}(x) = f_{\\hat{w}}(x)`} />. Just two
             notations for the same thing, but the second is more explicit in
             what is estimated!
+            <br />
+            <br />
+            We use <IM math={`RSS_{test}(\\hat{w})`} /> to mean the same thing
+            as <IM math={`RSS_{test}(\\hat{f})`} /> or{" "}
+            <IM math={`RSS_{test}(f_{\\hat{w}})`} />.
           </MarginNote>
-          :
+          , the test error is defined as:
         </p>
 
         <BM
           math={`RSS_{test}(\\hat{w}) = \\sum_{x_i \\in Test} \\left(y_i - f_{\\hat{w}}(x_i)\\right)^2`}
         />
 
+        <p>
+          More generally, a common definition of the test error is the average
+          loss for whichever loss function <IM math={`L`} /> you are using.
+        </p>
+
+        <BM
+          math={`error_{test}(\\hat{f}) = \\frac{1}{n_{test}}\\sum_{x_i \\in Test} L(y, \\hat{f}(x))`}
```

Comment:
Perhaps instead of using `n_test` which you haven't defined, use `|Test|` and add a footnote for cardinality?